03-Security-In-CICD/Readme.md

```diff
@@ -499,11 +499,13 @@ Along with input parameter, you can check for below options
 
 ![03_IP_Parameter_for_Task](../Images/03_IP_Parameter_for_Task_ARM.JPG)
 
-**Step-4**: Specify AzSKServerURL and EnableServerAuth variables if you want to use your own org-policy
+**Step-4**: Specify AzSKServerURL and EnableServerAuth variables if you want to use your own org-policy. Please follow this [link](../07-Customizing-AzSK-for-your-Org#how-does-azsk-use-online-policy) to understand how does AzSK use online policy.
 
 AzSKServerURL -> AzSK OnlinePolicyStoreUrl
 
 EnableServerAuth -> EnableAADAuthForOnlinePolicyStore
+
+Please follow [this](../07-Customizing-AzSK-for-your-Org#can-i-completely-override-policy-i-do-not-want-policy-to-be-run-in-overlay-method) if you want to completely override policy configuration.
```

Comment:
This line is not fitting into context of configuring AzSKServerURL. 

Generally we give some context and then reference link. Here we are directly giving link for overriding policy but current context is related to end user who is adding policy url for his pipeline. We can add ARM policy scenarios in contoso doc with some example of overriding ARM policies like adding new one/ deleting one etc.

Reply:
made the changes